src/atoms/SvgIcon/default/Egg.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 
+import Svg from './Svg';
+
 const Egg = props => (
-	<svg width="25px" height="30px" viewBox="0 0 25 23" version="1.1">
+	<Svg width="25px" height="30px" viewBox="0 0 25 23" version="1.1">
```

Comment:
why do we need to specify version in this svg? :)

Reply:
if it's necessary for all svg's then it could be moved to Svg component. If it's not necessary, it could be removed. How do you think?

Reply:
Should definitely be removed. However, there was so many tweaks that I wanted to just do changes that were related to each other in this PR, for better readability.

Reply:
Ok, closing comment then